hoomd/md/bond.py

```diff
@@ -46,8 +46,7 @@ class Harmonic(Bond):
 
         V(r) = \frac{1}{2} k \left( r - r_0 \right)^2
 
-    where :math:`\vec{r}` is the vector pointing from one particle to the other
-    in the bond.
+    where :math:`r` is the distance from one particle to the other in the bond.
```

Comment:
This formula uses `r` as a distance, so we don't need to define or introduce the vector quantity / direction. Also applies below.

Reply:
Yes, we can make this change here.